client/tests/auth.spec.ts

```diff
@@ -0,0 +1,118 @@
+import test, { expect } from "@playwright/test";
+
```

Comment:
Try to randomize the strings you use for passwords. You can use [randomstring](https://www.npmjs.com/package/randomstring) or similar tools. You can also develop your own mechanism.
The goal is to have less pr-built tests.